.gitignore

```diff
@@ -46,4 +47,5 @@ govuk-react-*.tgz
 !.yarn/patches
 !.yarn/plugins
 !.yarn/releases
-!.yarn/versions
\ No newline at end of file
+!.yarn/versions
+yarn.lock
```

Comment:
please can you revert the changes to this file?

Reply:
thanks will do